### PR TITLE
doc suggestions

### PR DESCRIPTION
**Pull request by glfejer**
This still has the enf0.conf changes, which are not needed.

### CODE REVIEW

File: enftun/enf0.conf [hunk 1]
```diff
@@ -5,7 +5,7 @@ version = "1.0"
 # TUN device settings
 tun : {
   # The name of the TUN device to use.
-  # dev = "enf0"
```
Comment (drbild) [inline enftun/enf0.conf]: This is the default value; no need to uncomment.

File: enftun/enf0.conf [hunk 2]
```diff
@@ -22,7 +22,7 @@ remote : {
   hosts = [ "23.147.128.112" ]
 
   # The listening port on the remote server.
-  # port = "443"
```
Comment (drbild) [inline enftun/enf0.conf]: This is the default value; no need to uncomment.

**Review by drbild — CHANGES_REQUESTED** [review]

File: enftun/README.md [hunk 3]
```diff
@@ -20,10 +20,18 @@ FROM xaptum:enftun
...
-To run this image in a local container, run:
-```
-make shell
-```
+To run this image in a local container:
+- Create a subdirectory named `enf0`
+    ``` 
+    mkdir enf0
+    ```
+- Copy the key files generated using the  [enf-services
```
Comment (zanebeckwith) [inline enftun/README.md]: To be more explicit, rather than `key files` how about `the private key and certificate`.

File: enftun/README.md [hunk 4]
```diff
@@ -20,10 +20,18 @@ FROM xaptum:enftun
...
+README](https://github.com/xaptum/enf-services) into `enf0/`
```
Comment (zanebeckwith) [inline enftun/README.md:29]: And, how about `...generated using the instructions in [enf...`?

**Review by zanebeckwith — CHANGES_REQUESTED** [review]
Thanks for this improvement, and for sussing out that the published image hadn't been updated since the change to use the enftun-packaged version of the CA cert file!

As David mentioned, no changes to the config are necessary (that was a result of your triaging the published image being old, right? That's now fixed)

Also, would you remove the commit about removing tools from the image, and make sure those changes aren't made to the `Dockerfile`? (Again, I'm guessing those things are artifacts from your testing)

**Review by drbild — APPROVED** [review]

**Comment by drbild** [comment]
@zanebeckwith 

A separate issue, but I think we should start versioning these images.    Thoughts?

**Review by zanebeckwith — APPROVED** [review]
Good change, and thanks for being the guinea pig!

**Comment by zanebeckwith** [comment]
@drbild 


> I think we should start versioning these images.

Agreed.

The `enf-apt-mirror` isn't currently versioned, right? So if there's any consistent versioning scheme we want to follow for all these images, we can specify that now.

Do you want to open an issue for adding versioning?

**Pull request merged**